### PR TITLE
Update showtime.ts

### PR DESCRIPTION
**Pull request by aneri18**
Update all Showtme descriptions to include partner name

### CODE REVIEW

File: src/config/partners/showtime.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ Wrap It Up with Incredible Raffles: Stick around to the end, and you could score
     {
       image: 'https://assets.onchainsummer.xyz/showtime-drop-4.mp4',
       creator: '0xDB83ea3B2A65300078532966Fd48322518632EE4',
-      name: `"Dad" Demo Access Pass`,
+      name: `Annika Rose: "Dad" Demo Access Pas`,
```
Comment (AshishYelekar) [inline src/config/partners/showtime.ts:77]: There's a typo on the "Pass" here

File: src/config/partners/showtime.ts [hunk 2]
```diff
@@ -114,7 +114,7 @@ Wrap It Up with Incredible Raffles: Stick around to the end, and you could score
     {
       image: 'https://assets.onchainsummer.xyz/showtime-drop-7.mp4',
       creator: '0x6AcDf8958f7852720733A375e6accFa8Ec7C42d0',
-      name: `Performance Access Pass`,
+      name: `Talia Lahoud: Performance Access Pass`,
```
Comment (AshishYelekar) [inline src/config/partners/showtime.ts:117]: I think "Access" is supposed to be removed based on the slack feedback?

File: src/config/partners/showtime.ts (hunk 1 above)
Comment (aneri18) [reply src/config/partners/showtime.ts:77]: will change

File: src/config/partners/showtime.ts (hunk 2 above)
Comment (aneri18) [reply src/config/partners/showtime.ts:117]: Thought that was an omission on our side. will check

**Review by AshishYelekar — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for AshishYelekar @ 2023-08-18 04:48:08 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Pull request merged**